docs/standard/microservices-architecture/multi-container-microservice-net-applications/multi-container-applications-docker-compose.md

```diff
@@ -440,30 +441,30 @@ A Dockerfile like this will work. However, you can substantially optimize your i
 
 In the container and microservices model, you are constantly starting containers. The typical way of using containers does not restart a sleeping container, because the container is disposable. Orchestrators (like Kubernetes and Azure Service Fabric) simply create new instances of images. What this means is that you would need to optimize by precompiling the application when it is built so the instantiation process will be faster. When the container is started, it should be ready to run. You should not restore and compile at run time, using `dotnet restore` and `dotnet build` commands from the dotnet CLI that, as you see in many blog posts about .NET Core and Docker.
 
-The .NET team has been doing important work to make .NET Core and ASP.NET Core a container-optimized framework. Not only is .NET Core a lightweight framework with a small memory footprint; the team has focused on optimized Docker images for three main scenarios and published them in the Docker Hub registry at <span class="underline">microsoft/dotnet</span>, beginning with version 2.1:
+The .NET team has been doing important work to make .NET Core and ASP.NET Core a container-optimized framework. Not only is .NET Core a lightweight framework with a small memory footprint; the team has focused on optimized Docker images for three main scenarios and published them in the Docker Hub registry at *microsoft/dotnet*, beginning with version 2.1:
 
-1.  **Development**: Where the priority is the ability to quickly interate and debug changes and size is secondary.
+1. **Development**: Where the priority is the ability to quickly iterate and debug changes, and where size is secondary.
```

Comment:
I'm not sure that iterate is the right word here. Integrate seems to me to make more sense. @mvelosop, @CESARDELATORRE, is iterate correct?

Reply:
"iterate" is correct, this is an extract from the [Official .NET Docker images page](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/net-core-net-framework-containers/official-net-docker-images#during-development-and-build)